plaso/cli/extraction_tool.py

```diff
@@ -186,6 +208,19 @@ def _PreprocessSources(self, extraction_engine):
 
     logger.debug('Preprocessing done.')
 
+  def _ReadParserPresetsFromFile(self):
```

Comment:
ReadPresetsFromFile() can also result in a uncaught MalformedPresetError from _ReadPresetsFromFileObject() that would be nice to catch here and raise as BadConfigOption as well - please add this functionality.

Reply:
ack